.github/workflows/ci.yaml

```diff
@@ -110,6 +110,12 @@ jobs:
           fetch-depth: 0
           submodules: true
 
+      - name: Install quilt
+        run: sudo apt update && sudo apt install quilt
```

Comment:
question: Do we need `-y` here?

Reply:
Huh, I guess not!  My guess is that it detects it is not an interactive terminal so it skips the question.